src/fides/api/models/privacy_request.py

```diff
@@ -189,6 +198,21 @@ def generate_request_callback_jwe(webhook: PolicyPreWebhook) -> str:
     )
 
 
+def generate_request_callback_pre_approval_jwe(webhook: PreApprovalWebhook) -> str:
+    """
+    Generate a JWE to be used to mark privacy requests as eligible / not-eligible for pre approval.
+    """
+    jwe = WebhookJWE(
+        webhook_id=webhook.id,
+        scopes=[PRIVACY_REQUEST_REVIEW],
+        iat=datetime.now().isoformat(),
+    )
+    return generate_jwe(
+        json.dumps(jwe.dict()),
+        CONFIG.security.app_encryption_key,
+    )
```

Comment:
ah okay, the main difference is the scope! it's nice we already have a matching one.